### ISSUE TITLE
[enhancement] Add more Image Transformation like the ones in keras

### ISSUE THREAD

**Issue by ZhongBaby** [deeplearning4j/DataVec#152]
I found that [image transformations in keras](https://keras.io/preprocessing/image/#imagedatagenerator) is versatile.
And I want to see that dl4j-datavec has at least as many as keras has.
Thanks!

**Comment by saudet** [comment]
Could we have list, in order of priority, if possible? Thanks!

**Comment by ZhongBaby** [comment]
e.g. rotation, shift.

**Comment by saudet** [comment]
We can already do that with WarpImageTransform. It's just random transformations, right? Or do you know of cases when specifying the magnitude of random shifts and rotations separately leads to higher accuracy?

**Comment by ZhongBaby** [comment]
Sorry, I see, I think after this [150](https://github.com/deeplearning4j/DataVec/issues/150) issue we don't need more image transforms.

**Comment by saudet** [comment]
Actually, it would make sense to rotate images by large angles, but without too much sheer, to gain some degree of rotation invariance. So I added RotateImageTransform for that, enjoy!
